e2e_tests/tests/experiment/experiment.py

```diff
@@ -360,9 +360,18 @@ def report_failed_experiment(experiment_id: int, state: str) -> None:
     )
 
     for trial in error_trials:
-        print("******** Start of logs for trial {} ********".format(trial["id"]), file=sys.stderr)
-        print("".join(trial_logs(trial["id"])), file=sys.stderr)
-        print("******** End of logs for trial {} ********".format(trial["id"]), file=sys.stderr)
+        print_trial_logs(trial["id"])
+
+
+def report_failed_trial(trial_id: int, state: str) -> None:
+    print(f"Trial {trial_id} was not COMPLETED but {state}", file=sys.stderr)
```

Comment:
non-blocking: not sure we need this other print

Reply:
just added so if you were reading the logs top to bottom there would be some explanation why there are about to be 100s of lines of trial logs before they get printed. and the separate func was just so the print didn't look as weird right next to pytest.fail failing with the same message 😆